docs/generic_geometry.ipynb

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
<p>I'd suggest leaving autoreload out of the examples - it can be handy for development, but it's a distraction in the docs.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/European-XFEL/EXtra-geom/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
My bad, sorry. Removed.